src/applications/coronavirus-research/sign-up/tests/reviewAndSubmit.cypress.spec.js

```diff
@@ -3,16 +3,14 @@ import featureTogglesEnabled from './fixtures/toggle-covid-feature.json';
 
 describe('COVID-19 Research Form', () => {
   describe('when entering valid information and submitting the sign up form', () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
With `experimentalSessionAndOrigin` enabled, session data is reset before each test, so in order to get this test passing, the URL needs to be visited before each test, along with the feature toggle call being mocked

Reply:
Looks like a good change at any rate.